packages/components/dialog/src/Dialog.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from 'react'
+
+import { CloseButton } from './DialogCloseButton'
+import { Content } from './DialogContent'
+import { Description } from './DialogDescription' // aria-describedby
+import { Overlay } from './DialogOverlay'
+import { Portal } from './DialogPortal'
+import { Root, type RootProps } from './DialogRoot'
+import { Title } from './DialogTitle' // aria-labelledby
+import { Trigger } from './DialogTrigger'
+
+Trigger.displayName = 'Dialog.Trigger'
+Portal.displayName = 'Dialog.Portal'
+Overlay.displayName = 'Dialog.Overlay'
+Content.displayName = 'Dialog.Content'
+CloseButton.displayName = 'Dialog.CloseButton'
+Title.displayName = 'Dialog.Title'
```

Comment:
Wouldn't we wrap the title and description from content within a `Heading`component, for ex.? Same for content itself...?

Reply:
It is planned I just created the ticket for this :) https://github.com/adevinta/spark/issues/1101

Basically `Popover.Header` will contain the `Popover.Title` but can also contain other elements. So I think we need the 2 of them.

You can also wrap a title with `VisuallyHidden` when you don't need a Header: https://www.radix-ui.com/docs/primitives/components/dialog#title

So I think we need both :) But Radix does not offer Header/Body/Footer that we need, So I created a ticket.

Reply:
Oh great! Even if I'm a bit afraid now, of the amount of subcomponents we would end with :stuck_out_tongue_closed_eyes: